airbyte-db/lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_39_1_001__CreateStreamResets.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.currentOffsetDateTime;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: update migration description in the class name
+public class V0_39_1_001__CreateStreamResets extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_39_1_001__CreateStreamResets.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    final DSLContext ctx = DSL.using(context.getConnection());
+    createResetsTable(ctx);
+  }
+
+  private static void createResetsTable(final DSLContext ctx) {
+    final Field<UUID> id = DSL.field("id", SQLDataType.UUID.nullable(false));
+    final Field<UUID> connectionId = DSL.field("connection_id", SQLDataType.UUID.nullable(false));
+    final Field<String> streamName = DSL.field("stream_name", SQLDataType.CLOB.nullable(false));
```

Comment:
We probably should place a uniqueness constraint on these two columns, since I don't believe we ever want to store multiple records in this table that both have the same connection ID and stream name.

Reply:
good call.